mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -1100,14 +1100,45 @@ class LogisticRegressionModel private[spark] (
 
   private lazy val _intercept = interceptVector.toArray.head
```

Comment:
Maybe this can use _interceptVector now rather than .toArray again? or at least just read off its first element directly